astro/src/content/docs/release-notes/index.mdx

```diff
@@ -892,7 +893,8 @@ This migration will add an index to the `identity_provider_links` table. It is n
   * Resolves [GitHub Issue #2118](https://github.com/FusionAuth/fusionauth-issues/issues/2118)
 
 ### Enhancements
-* Add default configuration for read and connect timeouts to the SMTP server configuration. This helps protect FusionAuth against an SMTP server that never closes a socket. From time to time we observed an SMTP server hold open a socket, and tie up a send thread which may block other senders.
+* Add default configuration for read and connect timeouts to the SMTP server configuration. This helps protect FusionAuth against an SMTP server that never closes a socket. From time to time we observed an SMTP server hold open a socket, and tie up a send thread which may block other senders. This includes the `mail.smtp.timeout` and 
+`mail.smtp.connectiontimeout` settings. The default value is `2000` for each. These may be overridden by navigating to the <Breadcrumb>Tenants -> Your Tenant -> Advanced</Breadcrumb> tab and adding them to the <Breadcrumb>SMTP settings</Breadcrumb> section.
```

Comment:
```suggestion
`mail.smtp.connectiontimeout` settings. The default value is `2000` for each. These may be overridden by navigating to the `Tenants -> Edit -> Advanced` tab and adding them to the `SMTP settings` Additional properties.
```

Reply:
Why no `<Breadcrumb>` @bhalsey ?

Reply:
We don't use `<Breadcrumb>` in this file